src/main/java/net/sf/jabref/logic/layout/LayoutEntry.java

```diff
@@ -644,9 +646,14 @@ private LayoutFormatter getLayoutFormatterByName(String name) throws Exception {
                             int startParam = i;
                             i++;
                             boolean escaped = false;
-                            while (((i + 1) < c.length) && !(!escaped && (c[i] == '"') && (c[i + 1] == ')'))) {
+                            while (((i + 1) < c.length)
```

Comment:
Yes, there is brace checking and handling code in `BibtexParser`, but that is absolutely not extractable. It depends on the internal usage of the `PushBackReader` in the class and interferes with parsing in general.